src/taskcollector-TaskCollector.ts

```diff
@@ -208,6 +208,11 @@ export class TaskCollector {
         editor.setValue(result.join("\n"));
     }
 
+    removeCheckboxFromLine(line: string): string {
+        const lineWithoutCheckbox = line.replace(/(?<=([-] ))\[.\] /, '');
```

Comment:
Did you test this on mobile? I had to change remove/change another regex I was using because it barfed on mobile. (I can not remember exactly how it failed, but I know that it did).